### ISSUE TITLE
Screenreader is not announcing combo box “select the folder that will contain your folder name”.

### ISSUE THREAD

**Issue by Yash14j** [microsoft/vscode#108297]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yMAS;#A11yTCS;#Win32;#DesktopApp;#A11y_VSCode;#WCAG1.3.1;#A11ySev2;#CAITestPass_May2020;#Benchmark;#Visual Studio Code;

**Environment Details:**
Application Name: Azure Extensions for VS Code
Version: 1.46.1 (user setup)
Commit: cd9ea6488829f560dc949a8b2fb789f3cdc05f5d
Date: 2020-06-17T21:13:20.174Z
Electron: 7.3.1
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
Windows Version: Windows10

**Repro Steps:**

1. Open Visual studio code
2. Click on extension button and search for "Azure Functions"
3. Click on install and move to the "Azure" button
4. Open screenreader and click on "Create New Project Button" from the toolbar
5. Create new project pop will appear
6. Press tab key and then press down arrow key.
7. Focus is on "browse". Now press "shift+tab key" to check the result

**Actual:**

When we press tab key and then again press shift+tab key screenreader does not announces name of combo box “select the folder that will contain your folder name”

This issue is also observed in Create Function and Deploy to function app screen
Same issue is also observed with both NVDA and JAWS.

**Expected:**

Screenreader should announce name of combo box “select the folder that will contain your folder name” when focus comes on it

**User Impact:**

User will not get proper information about the combo box

**Attachment:**
[#108297.zip](https://github.com/microsoft/vscode/files/5347331/108297.zip)

**Comment by JacksonKearl** [comment]
Issue here seems to be that when a quick pick is shown, the text input's description text does not get read, and instead the focused item in the list is read (even if focus is on the text input).
![recording (8)](https://user-images.githubusercontent.com/8586769/95640954-52589600-0a54-11eb-93bb-97aae5b65f99.gif)

**Comment by chrmarti** [comment]
/duplicate of https://github.com/microsoft/vscode/issues/104664

Please update to the latest version where this is fixed.
